features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -224,7 +227,8 @@ Feature: Collection
   #           offers: SGA (JS, TE), S&H (JS, TE, Foo)
   When I log out
     And I am logged in as "myname3"
-  When I go to the collections page
+  When all indexing jobs have been run 
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.